src/deploy/functions/deploy.ts

```diff
@@ -1,21 +1,38 @@
 import * as clc from "cli-color";
 import { setGracefulCleanup } from "tmp";
 
+import { checkHttpIam } from "./checkIam";
 import { functionsUploadRegion } from "../../api";
 import { logSuccess, logWarning } from "../../utils";
-import { checkHttpIam } from "./checkIam";
 import * as args from "./args";
-import * as gcp from "../../gcp";
+import * as backend from "./backend";
+import * as fs from "fs";
+import * as gcs from "../../gcp/storage";
+import * as gcf from "../../gcp/cloudfunctions";
+import { previews } from "../../previews";
+import { promise } from "ora";
```

Comment:
nit* I don't really get how we order our imports, but the `ora` import looks like it should be part of the first group of imports instead.